docs/INSTALL.md

```diff
@@ -1,130 +1,31 @@
 # Installation
 
-Open a `terminal` window and navigate to the folder where you want to
-install HADDOCK3; for example: `software`. The current installation
-instructions are local and will affect only your user.
-
-Before starting with the installation of HADDOCK3, make sure to properly [install CNS](CNS.md).
-If you have installed a previous version of HADDOCK, you may already have a suitable version of CNS.
-Please do [check your CNS installation](CNS.md#5-Check-installation) before proceeding.
-
-
-## 1. Clone this repository:
-
-Mind the `--recursive` flag when cloning!
-
-```bash
-git clone --recursive https://github.com/haddocking/haddock3.git
-```
-
-Go to the **haddock3** directory
-
-```bash
-cd haddock3
-```
-
-and type the following commands to first install the [FCC](https://github.com/haddocking/fcc) source code provided
-
-```bash
-cd src/fcc/src
-chmod u+x Makefile
-make
-cd -
-```
-
-Then, follow the same logic to install the [fast-rmsdmatrix](https://github.com/mgiulini/fast-rmsdmatrix) source code
-
-```bash
-cd src/fast-rmsdmatrix/src
-chmod u+x Makefile
-make fast-rmsdmatrix
-cd -
-```
-
-By the end of the above commands, you should be back to the **haddock3**
-main folder.
-
-## 2 Create a virtual environment with Python 3.9+ and install dependencies:
+Create a virtual environment with Python 3.10:
 
 You can use Python's `venv` or `conda` depending on your choice.
 Commands are provided below:
 
 ### with `venv`
 
 ```bash
-virtualenv venv --python=3.9
+virtualenv venv --python=3.10
```

Comment:
Why python3.10? We have always been using 3.9 (which is still quite distributed on various OS). Will it work with both 3.9 and 3.10? Has haddock3 been tested on 3.10?

Reply:
Yes

Reply:
Will address this in #1040